org.eclipse.xtext/src/org/eclipse/xtext/resource/EObjectAtOffsetHelper.java

```diff
@@ -157,4 +166,45 @@ protected EObject resolveCrossReferencedElement(INode node) {
 		return null;
 	}
 
+	private String getUnquotedToxenText(INode node) {
+		String text = NodeModelUtils.getTokenText(node);
+		if (text != null && text.startsWith("^")) {
+			return text.substring(1);
+		}
```

Comment:
is there a better way to do this unquoting?

Reply:
This is normally done by value converters